src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.unit.cs

```diff
@@ -549,11 +555,18 @@ public void FileSystemWatcher_Renamed()
             }
         }
 
-        [Fact]
+        [ConditionalFact]
```

Comment:
Normally we do this through a property, i.e. `[ConditionalFact(nameof(DangerousIOTestsEnabled))]`

Reply:
Yes, I know. But that removes ability to show custom text and you get generic skip message. 

I could certainly do that if that is preference.

Reply:
updated.